exercise-2/style.css

```diff
@@ -0,0 +1,72 @@
+.first {
+  width: 330px;
+  padding: 0px 0px 0px 30px;
+  margin: 25px 0px 0px 34px;
+  text-align: center;
+}
+.second {
+  width: 330px;
+  margin: -250px 0px 0px 438px;
+  text-align: center;
+}
+.three {
+  width: 330px;
+  padding: 0px 0px 0px 30px;
+  margin: 35px 0px 0px 26px;
+  text-align: center;
+}
+.four {
+  width: 330px;
+  margin: -234px 0px 0px 444px;
+  text-align: center;
+}
+
+li {
+  display: inline-block;
+  margin-right: 15px;
+}
+input {
+  visibility: hidden;
+}
+label {
+  cursor: pointer;
+}
+input:checked + label {
+  background: rgb(74, 175, 181);
+}
+
+.color {
```

Comment:
color is a very misleading class name here. Like, we are not applying any color-related property inside, why would we rename this as **color**? Please work on variable naming